src/setupTests.ts

```diff
@@ -3,3 +3,25 @@
 // expect(element).toHaveTextContent(/react/i)
```

Comment:
Can probably get rid of this comment now.

Reply:
cd2e1af